pysrc/scripts/sampling.py

```diff
@@ -81,6 +98,41 @@ def clean_results(results_folder: str) -> None:
     for tmp_file in result_files:
         os.remove(os.path.join(os.getcwd(), results_folder, tmp_file))
 
+def get_file_combinations(samples: list[tuple[str, str, float]], config: GlobalConfiguration) \
```

Comment:
```suggestion
def get_file_combinations(samples: list[tuple[str, str, float]]) \
```

It seems to be unused.

Reply:
would keep it for a bit, because i am not entirely sure how the method works with headers in the file.  Tested with WDC which contained header where the method worked as expected nevertheless